Modules/arraymodule.c

```diff
@@ -669,34 +805,176 @@ newarrayobject(PyTypeObject *type, Py_ssize_t size, const struct arraydescr *des
     return (PyObject *) op;
 }
 
+static inline int
```

Comment:
`static int` is fine. We don't need to mark these functions as `inline` outside of headers.